modelmapper/slack.py

```diff
@@ -0,0 +1,16 @@
+import json
+import requests
+
+
+def slack(text,
```

Comment:
@SkamDart Definitely will consider using it in future. It might be out of scope of this PR though. The current way of just hitting Slack's API for reporting works fine.